tap_five9/__init__.py

```diff
@@ -153,20 +153,21 @@ def do_sync(client, catalog, state, start_date):
 def main():
     # Parse command line arguments
     args = utils.parse_args(REQUIRED_CONFIG_KEYS)
-    client = Five9API(args.config)
+    config = args.config
+    client = Five9API(config)
 
     # If discover flag was passed, run discovery mode and dump output to stdout
     if args.discover:
-        catalog = discover(client, args.config.get('custom_reports'))
+        catalog = discover(client, config, config.get('custom_reports'))
```

Comment:
We pass the config down to the client. Instead of passing the `config` param to each of the functions, setting `self.config = config` and using `client.config` is probably easier. It's odd to me that we have to do something like this where we pass the config and then a key of the config.